dc/s2n-quic-dc/src/path/secret/map/cleaner.rs

```diff
@@ -68,7 +68,12 @@ impl Cleaner {
                     break;
                 }
                 state.cleaner().clean(&state, EVICTION_CYCLES);
-                let pause = rand::thread_rng().gen_range(5..60);
+                // in tests, we should try and be deterministic
+                let pause = if cfg!(test) {
+                    60
+                } else {
+                    rand::thread_rng().gen_range(5..60)
+                };
```

Comment:
We're sort of not deterministic anyway since this is wall clock time... but I guess this seems fine.

Reply:
For sure. But this is at least better than what we had before :smile:. The snapshots were quite flaky.